src/core/primitives.rs

```diff
@@ -72,6 +72,7 @@ pub fn setup(rng: &mut impl CryptoRngCore, tracing_level: usize) -> Result<Maste
 
     let mut tsk = TracingSecretKey::default();
     (0..=tracing_level).for_each(|_| tsk.increase_tracing(rng));
+
```

Comment:
Avoid adding unnecessary diffs (reading one own diffs prior to posting a MR is a good exercise and avoids letting things like that pass through).
```suggestion
```